app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt

```diff
@@ -129,17 +129,19 @@ class KyouenActivity : DaggerAppCompatActivity(), KyouenActivityHandlers {
         }
 
         stageRequest
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread())
-                .autoDisposable(scopeProvider)
-                .subscribe(
-                        { newStage ->
-                            stageModel = newStage
-                            showOtherStage(direction)
-                        },
-                        { throwable -> throw RuntimeException("I think, we are not called this.", throwable) },
-                        { loadNextStages(direction) }
-                )
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .autoDisposable(scopeProvider)
+            .subscribe(
+                { newStage ->
+                    stageModel = newStage
+                    showOtherStage(direction)
+                },
+                { throwable ->
+                    throw RuntimeException("I think, we are not called this.", throwable)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/f44fa79828c9d915587d9b3fabd8550e24815e45/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt#L141"></span>RuntimeException is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.</td>
    </tr>
  </tbody>
</table>